index.html

```diff
@@ -1302,15 +1379,15 @@ <h3>Expiration</h3>
         <h3>Status</h3>
 
         <p>
-Information about the current status of a <a>verifiable credential</a>, such
-as whether it is suspended or revoked, MAY be provided using the
-<code>credentialStatus</code> <a>property</a>.
+If information about the current status of a <a>verifiable credential</a>, such
+as whether it is suspended or revoked, is needed, then it MUST be provided
```

Comment:
This is impossible to test.

Reply:
Agreed, removed in 59ff3ba0d917ef2a6fcd10e9f14d04a3f7551942.